app/controllers/posts_controller.rb

```diff
@@ -387,13 +387,13 @@ def delete
       return
     end
 
-    if @post.post_type.is_freely_editable && !current_user&.is_moderator
+    if @post.post_type.is_freely_editable && !current_user&.at_least_moderator?
```

Comment:
Untested suggestion:

```suggestion
    if @post.post_type.is_freely_editable && current_user&.standard?
```

Reply:
Resolving due to `!user&.at_least_moderator?` and `user&.standard?` not being equal